modules/core/src/test/java/org/apache/ignite/testsuites/IgnitePerformanceTestSuite.java

```diff
@@ -0,0 +1,118 @@
+import org.apache.ignite.loadtests.direct.stealing.GridStealingLoadTest;
+import org.apache.ignite.loadtests.discovery.GridGcTimeoutTest;
+import org.apache.ignite.loadtests.dsi.cacheget.GridBenchmarkCacheGetLoadTest;
+import org.apache.ignite.loadtests.hashmap.GridBoundedConcurrentLinkedHashSetLoadTest;
+import org.apache.ignite.loadtests.hashmap.GridHashMapLoadTest;
+import org.apache.ignite.loadtests.job.GridJobExecutionSingleNodeLoadTest;
+import org.apache.ignite.loadtests.job.GridJobExecutionSingleNodeSemaphoreLoadTest;
+import org.apache.ignite.loadtests.job.GridJobLoadTest;
+import org.apache.ignite.loadtests.mergesort.GridMergeSortLoadTest;
+import org.apache.ignite.loadtests.nio.GridNioBenchmarkTest;
+import org.apache.ignite.marshaller.GridMarshallerPerformanceTest;
+import org.apache.ignite.spi.communication.tcp.GridTcpCommunicationSpiLanLoadTest;
+
+/**
+ * Tests suite for performance tests tests.
+ * Note: Most of these are resource-consuming or non-terminating.
+ */
+public class IgnitePerformanceTestSuite extends TestSuite {
+    /**
+     * @return Tests suite for orphaned tests (not in any test sute previously).
+     */
+    public static TestSuite suite() {
+        TestSuite suite = new TestSuite("Ignite Load-Test Suite");
+
+        suite.addTestSuite(GridCacheDhtPreloadPerformanceTest.class);
+        suite.addTestSuite(GridCacheIteratorPerformanceTest.class);
+        suite.addTestSuite(GridCacheMultiNodeLoadTest.class);
+        suite.addTestSuite(GridCacheConcurrentTxMultiNodeLoadTest.class);
+        suite.addTestSuite(GridCachePartitionedAffinityExcludeNeighborsPerformanceTest.class);
+        suite.addTestSuite(GridCachePartitionedAtomicLongLoadTest.class);
+        suite.addTestSuite(GridCacheWriteBehindStoreLoadTest.class);
+        suite.addTestSuite(GridCircularBufferPerformanceTest.class);
+        suite.addTestSuite(GridFuncPerformanceTest.class);
+        suite.addTestSuite(GridHashMapLoadTest.class);
+        suite.addTestSuite(GridLeanMapPerformanceTest.class);
+        suite.addTestSuite(GridMarshallerPerformanceTest.class);
+        suite.addTestSuite(GridMetadataAwareAdapterLoadTest.class);
+        suite.addTestSuite(GridMultiSplitsLoadTest.class);
+        suite.addTestSuite(GridMultiSplitsRedeployLoadTest.class);
+        suite.addTestSuite(GridSessionLoadTest.class);
+        suite.addTestSuite(GridSingleSplitsNewNodesMulticastLoadTest.class);
+        suite.addTestSuite(GridSingleSplitsRedeployLoadTest.class);
+        suite.addTestSuite(GridStealingLoadTest.class);
+        suite.addTestSuite(GridTcpCommunicationSpiLanLoadTest.class);
+        suite.addTestSuite(GridUnsafeMapPerformanceTest.class);
+        suite.addTestSuite(GridUnsafePartitionedMapPerformanceTest.class);
+        suite.addTestSuite(IgniteDataStreamerPerformanceTest.class);
+        suite.addTestSuite(SortedEvictionPolicyPerformanceTest.class);
+
+        // Main classes:
+        Class[] _$ = new Class[] {
```

Comment:
I suggest to add comment about purposes why these tests are placed into this array.